app/routes.js

```diff
@@ -62,6 +62,21 @@ module.exports = function(app) {
             .exec(resHandler(res))
     });
 
+    // get all neighbours
+    app.get('/api/sample/neighbours/:id', function(req, res) {
```

Comment:
and the images. Basically, there's no reason for these gets to be longer than one line.